.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -4,18 +4,24 @@ about: "Report confirmed bugs. For unconfirmed bugs, please visit our mailing li
 labels: "kind/bug/possible"
 
 ---
-
+<!--
 Please post all questions and issues on [janusgraph-users](https://groups.google.com/forum/#!forum/janusgraph-users)
 before opening a GitHub issue. Your questions will reach a wider audience there,
 and if we confirm that there is a bug, then you can open a new issue.
 
+Reminder check your current version and please update to a supported version of JanusGraph, 
+see https://docs.janusgraph.org/changelog/#currently-supported.
+-->
+
 Please include configurations and logs if available.
 
 For confirmed bugs, please report:
-- Version: 
-- Storage Backend: 
-- Mixed Index Backend:
+- Version: <!-- e.g.: 0.5.0,0.6.0 -->
+- Storage Backend: <!-- e.g.: cql, hbase, inmemory -->
+- Mixed Index Backend: <!-- e.g.: elasticsearch, none -->
 - Mailing list Thread URL:
```

Comment:
Do you think it would make sense to add Gitter Chat URL as well?
For example, I just created an issue with gitter link: https://github.com/JanusGraph/janusgraph/issues/2261